Uplift/Controllers/OnboardingViewController.swift

```diff
@@ -12,125 +12,227 @@ import UIKit
+            make.size.equalTo(CGSize(width: view.frame.width, height: 2))
+        }
+
+        runningPersonView.contentMode = .scaleAspectFit
+        runningPersonView.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 64 * vertScaling, height: 64 * vertScaling))
         }
     }
 
-    private func configureSlides() {
-        // slide images
-        let images = [UIImage(named: ImageNames.onboarding1), UIImage(named: ImageNames.onboarding2), UIImage(named: ImageNames.onboarding3), UIImage(named: ImageNames.onboarding4)].map { image -> Content in
-            let imageView = UIImageView(image: image)
-            imageView.contentMode = .scaleAspectFill
-            imageView.snp.makeConstraints { make in
-                make.height.width.equalTo(214)
-            }
-            let position = Position(left: 0.5, top: 0.47)
-            return Content(view: imageView, position: position)
+    private func setupSlides() {
+        // Onboarding View without TableView
+        let slidesTitleEnding = [viewSlides[0], viewSlides[3]].map { view -> Content in
+            let position = Position(left: 0.5, top: 0.45 * horizScaling)
+            return Content(view: view, position: position)
+        }
+        // Onboarding View with TableView
+        let slidesGymsClasses = viewSlides[1...2].map { view -> Content in
+            let position = Position(left: 0.5, top: 0.52 * horizScaling)
+            return Content(view: view, position: position)
         }
+        let contentSlides = [
+            slidesTitleEnding[0], slidesGymsClasses[0], slidesGymsClasses[1], slidesTitleEnding[1]
+        ]
 
-        // dismiss button
-        let button = UIButton()
-        button.frame = CGRect(x: 0, y: 0, width: 235, height: 64)
-        button.setTitle(ClientStrings.Onboarding.endOnboarding, for: .normal)
-        button.addTarget(self, action: #selector(dismissOnboarding), for: .touchUpInside)
-        button.titleLabel?.font = ._14MontserratBold
-        button.setTitleColor(UIColor.primaryBlack, for: .normal)
-
-        button.backgroundColor = .primaryYellow
-        button.layer.cornerRadius = 32
-        button.layer.shadowOffset = CGSize(width: 0, height: 4)
-        button.layer.shadowColor = UIColor.buttonShadow.cgColor
-        button.layer.shadowOpacity = 0.5
-        let buttonPosition = Position(left: 0.5, top: 0.71)
-        let startButton = Content(view: button, position: buttonPosition, centered: true)
-
-        // add content to slides
         var slides = [SlideController]()
-
-        for index in 0..<images.count {
-            var contents = [images[index]]
-            if index == images.count - 1 {
-                contents.append(startButton)
-            }
-            let controller = SlideController(contents: contents)
-            slides.append(controller)
+        contentSlides.forEach {
+            slides.append(SlideController(contents: [$0]))
         }
```

Comment:
can simplify to `let slides = contentSlides.map { SlideController(contents: [$0]) }`